helm-chart/splunk-operator/values.yaml

```diff
@@ -0,0 +1,137 @@
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  # Add annotations to Splunk Operator deployment
+  annotations: {}
+
+  # Add pod annotations to Splunk Operator deployment pod
+  podAnnotations: {}
+
+  # Set security context for manager container, default ensures that no child process of manager can gain more privileges than manager
+  # reference: https://kubernetes.io/docs/concepts/security/pod-security-policy/#privilege-escalation
+  containerSecurityContext:
+    allowPrivilegeEscalation: false
+
+  # Set security context for Splunk Operator pod 
+  # reference: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#podsecuritycontext-v1-core
+  securityContext:
+    runAsUser: 1001
+    fsGroup: 1001
+    runAsNonRoot: true
+
+  # Set usage of host networking and namespace
+  # reference: https://kubernetes.io/docs/concepts/security/pod-security-policy/#host-namespaces
+  hostNetwork: false
+  hostPID: false
+  hostIPC: false
+
+  # Define liveness probe to check if manager container is running
+  # reference: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: 8081
+    initialDelaySeconds: 15
+    periodSeconds: 20
+
+  # Define readiness probe to check if manager container is ready for requests
+  # reference: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe
+  readinessProbe:
+    httpGet:
+      path: /readyz
+      port: 8081
+    initialDelaySeconds: 5
+    periodSeconds: 10
+
+  # Set manager service http port and define additional ports if necessary
+  # reference: https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/
+  service:
+    type: ClusterIP
+    httpPort: 80
+    additionalPorts: []
+  # - name: metric
+  #   port: 8080
+  #   protocol: TCP
+  #   targetPort: 8080
+  
+  # Set resource requests and limits for manager container
+  resources:
+    limits:
+      cpu: 200m
+      memory: 100Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+
+  # Specify nodes that can schedule the Splunk Operator pod
+  # reference: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
+  nodeSelector: {}
+
+  # Specify Splunk Operator pod tolerations
+  # reference: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+  tolerations: []
+
+  # Define affinity scheduling rules
+  # reference: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#affinity-v1-core
+  affinity: {}
+
+  # Set duration in seconds the pod needs to terminate gracefully
+  # reference: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#pod-v1-core
+  terminationGracePeriodSeconds: 10
+
+  # Specify volumes for Splunk Operator pod
+  # refernece: https://kubernetes.io/docs/concepts/storage/volumes/
+  volumes: []
```

Comment:
we have volume mount for splunk-operator also volume , check master branch or develop branch

Reply:
+1

Reply:
Added app volume and volume mount.